src/app/shared/featured-sale-offers/sale-offers-adidas/sale-offers-adidas.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, computed, inject, Input } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { RouterModule } from '@angular/router';
+import { SaleOfferService } from '../../../modules/offers/sale-offers/services/sale-offer.service';
+import { SaleOfferCardHomeComponent } from '../../../modules/offers/sale-offers/sale-offer-card-home/sale-offer-card-home.component';
+
+@Component({
+  selector: 'app-sale-offers-adidas',
+  imports: [
+    CommonModule,
+    RouterModule,
+    SaleOfferCardHomeComponent
+  ],
+  templateUrl: './sale-offers-adidas.component.html',
+  styleUrls: ['./sale-offers-adidas.component.scss'],
+})
+export class SaleOffersAdidasComponent {
+  @Input() brandIdFilter: number = 4; 
+  private saleOfferService = inject(SaleOfferService);
+
+  offers = computed(() => {
```

Comment:
Wiesz, że to nie będzie działać,

Domyślnie zwracane jest pierwsze 50 ofert, więc robisz filtrowanie na 50 zwrócony ofertach. Powinien pójśc strzał do API z parametrem brandId odpowiednim dla tego branda